joschahenningsen/TUM-Live
# Add video to audio transcoding

### Motivation and Context
Addresses #789 

### Description
Add ffmpeg command to transcode video to audio for an audio only watch mode.

### Steps for Testing
- 1 Admin

1. Log in
2. Upload VOD
3. In `cfg.StorageDir` should be a .m4a file next to a video file.
